Assets/MixedRealityToolkit.Services/DiagnosticsSystem/MixedRealityToolkitVisualProfiler.cs

```diff
@@ -62,6 +69,12 @@ public bool FrameInfoVisible
             set { frameInfoVisible = value; }
         }
 
+        private bool ShouldShowFrameInfo => frameInfoVisible
+#if WINDOWS_UWP
+            && (appCapture == null || !appCapture.IsCapturingVideo)
```

Comment:
Nit: We could remove the checks here since ShouldShowProfiler handles the global visibility of the profiler. I.E. If ShouldShowProfiler is false ShouldShowFrameInfo doesn't matter since the whole window is invisible.

Reply:
oh! okay cool!

Reply:
Reverted